ec3730/Controllers/PingViewController.swift

```diff
@@ -307,7 +307,7 @@ struct PingSwiftUIViewController: View {
                                     }
                                 })
                             }.padding()
-                        }.background(Color.black.ignoresSafeArea(.all, edges: .horizontal)).onTapGesture(perform: {
+                        }.background(Color.black).padding(.top, 1).onTapGesture(perform: {
```

Comment:
Why is the padding necessary at all?

Reply:
Embedding  scroll view in navigation and giving it a background causes such issues and it's quite famous issue if you see. There are many work around which works in some cases depending on your subviews. I tried those and then I went for simplest solution putting a view between navigation and scroll and it solve the overlapping and navigation view flickering issue. I replaced the empty view with padding of 1. Then saw on internet that other people are also using this approach of using empty view.